File: Wizzi/Dtos/Clientes/LocalizacionDto.cs
using Wizzi.Dtos.Localizaciones;

namespace Wizzi.Dtos.Clientes
{
    public class LocalizacionDto
    {
        public VerPaisDto Pais { get; set; }
        public VerProvinciaDto Provincia { get; set; }
        public VerCantonDto Canton { get; set; }
        public VerParroquiaDto Parroquia { get; set; }
    }
}

File: Wizzi/ClientApp/src/dtos/Clientes/localizacionDto.ts
import { VerPaisDto, VerProvinciaDto, VerCantonDto, VerParroquiaDto } from "../Localizaciones";

export interface LocalizacionDto {
    pais: VerPaisDto;
    provincia: VerProvinciaDto;
    canton: VerCantonDto;
    parroquia: VerParroquiaDto;
}